ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -109,7 +109,9 @@ def _parse_data_model(self):
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
                     description = mnode.get("description")
-                    dbt_owner = cnode["metadata"].get("owner")
+                    dbt_owner = mnode["meta"].get("owner") or cnode["metadata"].get(
+                        "owner"
+                    )
```

Comment:
Not sure if this condition is possible, but could it be the case where `mnode["meta"]` or `cnode["metadata"]` is `None`? If so, should we add a check?

Reply:
@nahuelverdugo the `meta` and `metadata` keys will always be present as a part of DBT schema